helm-chart/renku/values.yaml

```diff
@@ -1464,18 +1498,18 @@ dataService:
     create: true
   image:
     repository: renku/renku-data-service
-    tag: "0.49.0"
+    tag: "0.50.0"
     pullPolicy: IfNotPresent
   k8sWatcher:
     image:
       repository: renku/data-service-k8s-watcher
-      tag: "0.49.0"
+      tag: "0.50.0"
       pullPolicy: IfNotPresent
     resources: {}
   dataTasks:
     image:
       repository: renku/data-service-data-tasks
-      tag: "0.49.0"
+      tag: "0.50.0"
```

Comment:
Already available in the release-2.5.0 branch

Reply:
This is because the branch was not rebased properly. It is resolved now.